src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,119 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class FileEditor {
+    private Scanner scanner = new Scanner(System.in);
+    private FileCreator fileCreator = new FileCreator();
+    private FileReader fileReader = new FileReader();
+
+    public void start() {
+        boolean work = true;
+        infoMessage();
+        while (work) {
+            String[] variant = scanner.nextLine().split(" ");
+            checkInputLength(variant.length);
+            switch (variant[0].toLowerCase()) {
+                case "create":
+                    if (variant.length != 3) {
+                        break;
+                    }
+                    System.out.print(fileCreator.create(variant[1], variant[2]));
+                    break;
+                case "read":
+                    if (variant.length != 3) {
+                        break;
+                    }
+                    System.out.print(fileReader.read(variant[1], variant[2]));
+                    break;
+                case "info":
+                    if (variant.length != 3) {
+                        break;
+                    }
+                    System.out.print(fileReader.info(variant[1], variant[2]));
+                    break;
+                case "help":
+                    if (variant.length == 2) {
+                        System.out.println(help(variant[1]));
+                        break;
+                    }
+                    help();
+                    break;
+                case "exit":
+                    work = false;
+                    break;
+                default:
+                    wantToSave();
+            }
+        }
+    }
+
+    private void checkInputLength(int inputLength) {
+        if (inputLength > 3 || inputLength < 1) {
+            infoMessage();
+        }
+    }
+
+    private void infoMessage() {
+        System.out.println("Please enter valid command with correct amount parameters.\n"
+                + "You can get list of available commands by entering 'help'.\n>");
+    }
+
+    private void help() {
+        System.out.println("create [path] [filename] - creates new file at path\n"
+                + "read [path] [filename] - reads file at path\n"
+                + "info [path] [filename] - shows info on file at path\n"
+                + "help - shows all commands\n"
+                + "help [command] - shows more detailed info on the command\n"
+                + "exit - exits program\n");
+    }
+
+    private String help(String command) {
+        HashMap<String, String> hashMap = new HashMap<>();
```

Comment:
let's make this map a static class field
and fill it in static initialization block https://www.geeksforgeeks.org/g-fact-79/